demos/common/python/models/yolo.py

```diff
@@ -143,6 +143,33 @@ def _parse_yolo_region(predictions, input_size, params, threshold, multiple_labe
                                          confidence.item(), class_id.item()))
         return objects
 
+    # For model which includes postprocess
+    @staticmethod
+    def _parse_yolo_with_postprocess_region(predictions, threshold, multiple_labels=True):
+        objects = list()
+        # ------------------------------------------- Parsing YOLO Region output ---------------------------------------
+        for prediction in predictions:
+            x, y, width, height, object_score = prediction[:5]
+            class_probabilities = prediction[5:]
+            if object_score < threshold:
+                continue
+
+            if multiple_labels:
+                for class_id, class_probability in enumerate(class_probabilities):
+                    confidence = object_score * class_probability
+                    if confidence > threshold:
```

Comment:
same threshold is used for both `object_score` and `confidence = object_score * class_probability`. Is that correct?

Reply:
I think so, because the same behavior of the original function `_parse_yolo_region`